relayer/src/event/monitor.rs

```diff
@@ -81,37 +105,51 @@ pub struct EventMonitor {
     rt: Arc<TokioRuntime>,
 }
 
+async fn run_driver(
+    driver: WebSocketClientDriver,
+    tx: mpsc::UnboundedSender<tendermint_rpc::Error>,
+) {
+    if let Err(e) = driver.run().await {
+        if tx.send(e).is_err() {
+            println!("failed to relay driver error to event monitor");
```

Comment:
I don't see where is the EventMonitor consumes (or handles) the errors which this method `send`s here.

Reply:
Also, is it better to use `error!` (which prints to stderr) here instead of `println!`?

Reply:
> Also, is it better to use error! (which prints to stderr) here instead of println!?
 
Right, that's a leftover from my spike which didn't use tracing.

Reply:
> I don't see where is the EventMonitor consumes (or handles) the errors which this method sends here.

https://github.com/informalsystems/ibc-rs/blob/31c992a46e40f11e0dfb6044f5520fdd0ca4bb3d/relayer/src/event/monitor.rs#L272

Reply:
d75bc22d